aws/ecs_cluster_test.go

```diff
@@ -11,18 +11,17 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-const region = "eu-west-1"
-
 // Test we can create a cluster, tag it, and then find the tag
 func TestCanTagEcsClusters(t *testing.T) {
 	t.Parallel()
+	region, err := getRandomRegion()
```

Comment:
yeah - I discovered why I don't get any notifications if I miss this out basically. It's because I keep naming my error `errz which it thinks is handled if I've handled it anywhere else. How do you deal with this? Do you name every error separately? (that's been my way around it now)

Reply:
TBH, I mostly just built muscle memory at this point where, any time I do `err := ...`, I immediately check if `err` is `nil`... But yea, different var names is one way. In some cases, where `err` is the only var on the left you  need, you can do:

```go
if err := someThingThatReturnsError(); err != nil {
  // handle the err
}
```

That way, the `err` is only scoped to that one if-statement, so even though it's the same name, it only exists there, and is checked as part of this one-liner.

Reply:
yep, I like this! Thank you, Jim!

Reply:
I think I'll just do my own pre-commit hook to make me double-check a few of these things until I get into a habit.